Flick/Assets.xcassets/plusCircle.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```

Comment:
I think it is good to start just adding the three sizes as we develop. Can we confirm the sizes for each so that we always know what sizes we are putting for 1x 2x and 3x? Like sometimes when I just export from Figma, I can't really tell what the dimensions actually are.

Reply:
https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/image-size-and-resolution/
I don't think there are specific dimensions required for 1x, 2x, 3x. It's just relative to each other. I just exported based on 1x, 2x, 3x on Figma. That should be fine I believe.

I think there's dimension requirement for app icon